scania-digital-design-system/sdds
# fix: various post theme removal issues

**Describe pull-request**  
Various fixes to enable components to load theming variables

**How to test**  
1. Components run a version of theme-light: 4.4.0 (no more component styling variables there)
2. Run over components in Storybook to see if all colors are available on every component
